packages/data-flow/src/internal.ts

```diff
@@ -5,7 +5,7 @@ export * from "./abis/index.js";
 export * from "./utils/index.js";
 export * from "./data-loader/index.js";
 export * from "./eventsFetcher.js";
-export * from "./strategyRegistry.js";
+export * from "./registries/index.js";
```

Comment:
will we add a `strategy` directory within registries in the future ? i mean, once we develop the events registry we will need it

Reply:
yeah that was what i had in mind but will refactor it now

Reply:
[aa3ee27](https://github.com/defi-wonderland/grants-stack-indexer-v2/pull/40/commits/aa3ee2776048258d2dccdc6b2be57f5dcdc5b8c1)